Description: Fix version number in Makefile
 Fix version number in Makefile and add "debian" suffix.
 .

Author: John Paul Adrian Glaubitz <glaubitz@physik.fu-berlin.de>
Last-Update: 2017-09-18

--- virtualjaguar-2.1.3.orig/Makefile
+++ virtualjaguar-2.1.3/Makefile
@@ -53,11 +53,11 @@ obj:
 
 prepare: obj
 	@echo -e "\033[01;33m***\033[00;32m Preparing to compile Virtual Jaguar...\033[00m"
-#	@echo "#define VJ_RELEASE_VERSION \"v2.1.4\"" > src/version.h
-#	@echo "#define VJ_RELEASE_SUBVERSION \"Final\"" >> src/version.h
-#	@echo "#define VJ_REMOVE_DEV_CODE" >> src/version.h
-	@echo "#define VJ_RELEASE_VERSION \"GIT `git log -1 --pretty=format:%ci | cut -d ' ' -f 1 | tr -d -`\"" > src/version.h
-	@echo "#define VJ_RELEASE_SUBVERSION \"2.1.4 Prerelease\"" >> src/version.h
+	@echo "#define VJ_RELEASE_VERSION \"v2.1.3\"" > src/version.h
+	@echo "#define VJ_RELEASE_SUBVERSION \"debian\"" >> src/version.h
+	@echo "#define VJ_REMOVE_DEV_CODE" >> src/version.h
+#	@echo "#define VJ_RELEASE_VERSION \"GIT `git log -1 --pretty=format:%ci | cut -d ' ' -f 1 | tr -d -`\"" > src/version.h
+#	@echo "#define VJ_RELEASE_SUBVERSION \"2.1.4 Prerelease\"" >> src/version.h
 
 virtualjaguar: sources libs makefile-qt
 	@echo -e "\033[01;33m***\033[00;32m Making Virtual Jaguar GUI...\033[00m"
